### PR TITLE
Implement new dashboard

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Implementing new dashboard layout  https://github.com/ooni/probe/issues/1094

All the changes are visual, no algorithms were changed, better test it on a simulator.

### CODE REVIEW

File: ooniprobe/View/DashboardTableViewController.m [hunk 1]
```diff
@@ -73,7 +102,7 @@ -(IBAction)run:(id)sender{
                                message:NSLocalizedString(@"Modal.Error.NoInternet", nil) inView:self];
 }
 
--(void)runAll{
+-(IBAction)runAll{
```
Comment (bassosimone) [inline ooniprobe/View/DashboardTableViewController.m:105]: Here you are not returning anything, yet you changed the return value. Why is that?

File: ooniprobe/View/TestResults/Rows/DashboardTableViewCell.m [hunk 2]
```diff
@@ -38,21 +41,53 @@ - (void)setHighlighted:(BOOL)highlighted animated:(BOOL)animated{
...
+//From https://stackoverflow.com/questions/8098130/how-can-i-tint-a-uiimage-with-gradient
+-(UIImage *)imageWithGradient:(UIImage *)img startColor:(UIColor *)color1 endColor:(UIColor *)color2 {
+    UIGraphicsBeginImageContextWithOptions(img.size, NO, img.scale);
+    CGContextRef context = UIGraphicsGetCurrentContext();
+    CGContextTranslateCTM(context, 0, img.size.height);
+    CGContextScaleCTM(context, 1.0, -1.0);
+
+    CGContextSetBlendMode(context, kCGBlendModeNormal);
+    CGRect rect = CGRectMake(0, 0, img.size.width, img.size.height);
+    //CGContextDrawImage(context, rect, img.CGImage);
```
Comment (bassosimone) [inline ooniprobe/View/TestResults/Rows/DashboardTableViewCell.m:75]: I suggest to remove commented out code

File: ooniprobe/View/DashboardTableViewController.m (hunk 1 above)
Comment (lorenzoPrimi) [reply ooniprobe/View/DashboardTableViewController.m:105]: `IBAction` is needed to connect a function in a storyboard object

File: ooniprobe/View/TestResults/Rows/DashboardTableViewCell.m (hunk 2 above)
Comment (lorenzoPrimi) [reply ooniprobe/View/TestResults/Rows/DashboardTableViewCell.m:75]: Right!

File: ooniprobe/View/DashboardTableViewController.m (hunk 1 above)
Comment (bassosimone) [reply ooniprobe/View/DashboardTableViewController.m:105]: I don't get why you aren't returning anything? Is there a default `return nil`?

Comment (bassosimone) [reply ooniprobe/View/DashboardTableViewController.m:105]: So, it turns out `IBAction` resolves to `void`

**Pull request merged**